keyboards/ztboards/after/config.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2019 
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x0927 // ZT Boards
+#define PRODUCT_ID   0x0001
+#define DEVICE_VER   0x0001
+#define MANUFACTURER ZTBoards
+#define PRODUCT      ZTBoards After
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 16
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { B3, F6, F5, D5, B2 }
+#define MATRIX_COL_PINS { D0, D1, D2, D3, D7, D6, D4, C7, C6, B6, B5, B4, F7, F0, F4, F1 }
+#define UNUSED_PINS
+#define ENCODERS_PAD_A { B1 }
+#define ENCODERS_PAD_B { B0 }
+#define ENCODER_RESOLUTION 4 
```

Comment:
```suggestion
```

Reply:
Why are we deleting this?

Reply:
It's the default: https://github.com/qmk/qmk_firmware/blob/master/quantum/encoder.c#L27

Reply:
Ah I'm a dummy. With the new changes, I'm not passing the PR Lint keyboards/ info check.

Reply:
That's fine, it's not for these boards.

Reply:
Okay! Am I good to go in regards to the checks?